Style.css

```diff
@@ -1,1045 +1,981 @@
+  margin-bottom: 10px;
+  margin-right: 24px;
+  display: flex;
 }
 
-.fa-icon {
+.social-media {
   display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  padding: 8px;
-  width: 40px;
-  height: 40px;
-  border-radius: 8px;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+  justify-content: space-between;
+  padding-top: 20px;
 }
 
-.fa-icon li {
-  margin-top: -76%;
-  margin-left: 25%;
-  display: inline;
+.icons {
+  width: 18px;
+  height: 20px;
+  margin: 10px;
 }
 
-.headline-2 {
-  width: 327px;
-  height: 83px;
-  font-family: "Crete Round", sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 40px;
-  line-height: 52px;
-  color: #1c1a19;
-  margin-top: 20%;
+.img-list {
+  display: inline;
 }
 
-.h-row {
-  /* Vector */
-
-  width: 327px;
-  height: 0;
-  border: 1px solid #212121;
+.icon-list {
+  list-style-type: none;
 }
 
-.img-1 {
-  width: 327px;
-  height: 237px;
-  background: url("images/img_1.png");
+.last-page-icons {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
-.headline-3 {
-  width: 327px;
-  height: 36px;
-  font-family: "Crete Round", sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 32px;
-  line-height: 44px;
-
-  /* or 138% */
-
+.b-img-1 {
   display: flex;
-  align-items: center;
-
-  /* Neutrals/N900 */
-
-  color: #091e42;
-
-  /* Inside auto layout */
-
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  justify-content: space-between;
+  height: 55px;
 }
 
-.text-content-1 {
-  width: 327px;
-  height: 147px;
-
-  /* Theme 3 / Body Reguler 2 */
-
-  font-family: "Inter", sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-
-  /* or 150% */
-
-  /* Neutrals/N900 */
-
-  color: #091e42;
-
-  /* Inside auto layout */
-
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+.desktop-b-img-1 {
+  display: none;
 }
 
-.btn-css {
-  box-sizing: border-box;
-
-  /* Auto layout */
-
+.footer-page {
   display: flex;
   flex-direction: row;
-  align-items: center;
-  padding: 10px 12px;
-  width: 55px;
-  height: 40px;
-
-  /* Neutrals/N0 */
-
-  background: #fff;
-
-  /* Neutrals/N90 */
-
-  border: 1px solid #8993a4;
-
-  /* Inside auto layout */
-
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  justify-content: space-between;
+  width: 100%;
 }
 
-.btn-css :hover {
-  font-size: medium;
-  color: #836464;
+.icons:hover {
+  border: 1px solid #ff6b00;
 }
 
-.btn-html {
-  box-sizing: border-box;
-
-  /* Auto layout */
-
+.my-proj {
+  width: 100%;
   display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 10px 12px;
-  width: 67px;
-  height: 40px;
-
-  /* Neutrals/N0 */
-
-  background: #fff;
-
-  /* Neutrals/N90 */
-
-  border: 1px solid #8993a4;
-
-  /* Inside auto layout */
-
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-
-.btn-html :hover {
-  font-size: medium;
-  color: #836464;
+  flex-direction: column;
+  align-items: flex-start;
```

Comment:
- [ ] As suggested by the previous reviewer, the project cards in your projects section are aligned to the left wide on the mobile display. they should be aligned in the center properly. I have the fix for you, you just need to change the line 145 to `align-items: center;` to make it centered.
your design
<img width="539" alt="Screenshot 2023-06-17 at 9 01 54 PM" src="https://github.com/MohammadYaser/Portfolio/assets/96683843/0cf1d1b5-9139-4e87-8511-d11d49e18cd8">